scripts/tf-provider-incap-orch.sh

```diff
@@ -0,0 +1,259 @@
+}
+
+install(){      
+    log_entry ${FUNCNAME[0]}
+    validation
+
+    if [ ! -f "`which go`" ]; 
+    then 
+        log_info "${FUNCNAME[0]}" "Installing Golang..."
+        brew install go; 
+    else
+        log_info "${FUNCNAME[0]}" "Golang detected"
+    fi;
+
+    if [ ! -f "`which git`" ];   
+    then 
+        log_info "${FUNCNAME[0]}" "Installing GIT..."
+        brew install git; 
+    else
+        log_info "${FUNCNAME[0]}" "GIT detected"
+    fi;
+
+    if [ ! -f "`which terraform`" ];   
+    then 
+        log_info "${FUNCNAME[0]}" "Installing Terraform local client..."
+        brew tap hashicorp/tap
+        brew install hashicorp/tap/terraform
+    else
+        log_info "${FUNCNAME[0]}" "Terraform detected"
+    fi;
+
+    log_info "${FUNCNAME[0]}" "Login user is ${USER}"
+
+    mkdir -p $GOPATH
+    chmod -R 777 $GOPATH
+    mkdir -p $TERRAFORM
+    chmod -R 777 $TERRAFORM
+
+    log_info "${FUNCNAME[0]}" "Terraform GIT path $PROVIDER_GIT_LOCAL"
+
+    if [ -d "$PROVIDER_GIT_LOCAL" ]; 
+    then 
+        log_info "${FUNCNAME[0]}" "Pulling repo terraform-provider-incapsula"
+        git --git-dir=$PROVIDER_GIT_LOCAL/.git config core.fileMode false
+        git --git-dir=$PROVIDER_GIT_LOCAL/.git pull
+    else 
+        log_info "${FUNCNAME[0]}" "Cloning repo terraform-provider-incapsula"
+        git clone $PROVIDER_GIT $PROVIDER_GIT_LOCAL
+    fi
+
+    log_info "${FUNCNAME[0]}" "Creating ~/.terraformrc file"
+    
+    echo "provider_installation {
+  filesystem_mirror {
+    path    = \"/Users/$USER/workspace/go/plugins\"
+    include = [\"registry.terraform.io/terraform-providers/incapsula\"]
+  }
+}" > ~/.terraformrc
+
+    PROVIDER_VERSION=`git --git-dir=$PROVIDER_GIT_LOCAL/.git tag | tail -1 | cut -c2-`
+    log_info "${FUNCNAME[0]}" "Git provider version $PROVIDER_VERSION"
+
+    mkdir -p $PROVIDER_PLUGIN/$PROVIDER_VERSION/darwin_amd64/
+    log_info "${FUNCNAME[0]}" "Provider binary path $PROVIDER_PLUGIN/$PROVIDER_VERSION/darwin_amd64/"
+
+    log_info "${FUNCNAME[0]}" "Building local provider..."
+    make -C $PROVIDER_GIT_LOCAL fmt && make -C $PROVIDER_GIT_LOCAL build    
+    rm -rf $PROVIDER_PLUGIN/$PROVIDER_VERSION/darwin_amd64/ ||:
+    cp -r $PROVIDER_GIT_LOCAL/ $PROVIDER_PLUGIN/$PROVIDER_VERSION/darwin_amd64/
+
+    if [ ! -f "$MAIN_TF" ]; 
+    then 
+        log_info "${FUNCNAME[0]}" "Created file $MAIN_TF"
+        echo "terraform {
+  required_providers {
+    incapsula = {
+      source = \"terraform-providers/incapsula\"
+      version = \"$PROVIDER_VERSION\"
+    }
+  }
+}
+ 
+variable \"incapsula_api_id\" {
+  type        = number
+  description = \"API ID\"
+}
+ 
+variable \"incapsula_api_key\" {
+  type        = string
+  description = \"API KEY\"
+}
+
+provider \"incapsula\" {
+  api_id = var.incapsula_api_id
+  api_key = var.incapsula_api_key
+  base_url = \"https://my.impervaservices.com/api/prov/v1\"
```

Comment:
change to prod URLs

Reply:
I've removed the URLS pointing to Stage -